charts/images.yaml

```diff
@@ -49,7 +49,7 @@ images:
 - name: csi-driver
   sourceRepository: github.com/kubernetes-sigs/aws-ebs-csi-driver
   repository: k8s.gcr.io/provider-aws/aws-ebs-csi-driver
-  tag: "v0.10.1"
+  tag: "v1.1.0"
```

Comment:
Can we wait for the [next](https://github.com/kubernetes-sigs/aws-ebs-csi-driver/pull/924#issuecomment-859771242) release planned next week?

Reply:
Okay, let's then wait for v1.1.1.

/hold

Reply:
There is no activity in the upstream for v1.1.1.

/hold
/remove lifecycle/stale

Reply:
/unhold